### DISCUSSION TITLE
llvmlite temporary fix for building with llvm-11

### DISCUSSION THREAD

**Discussion by revelator** [msys2/MINGW-packages#8386] [General]
this patch will allow building llvmlite with llvm-11until it is updated upstream.
[llvm-11.0.zip](https://github.com/msys2/MINGW-packages/files/6323956/llvm-11.0.zip)

not added to any pull request as it is allready planned for the next numba llvmlite releases, but if you need the functionality now this will take care of it in the meantime. 

P.s need llvmlite-0.36.0 for this patch.

**Comment by lazka** [comment]
thanks, https://github.com/msys2/MINGW-packages/pull/8419

**Reply by revelator** [reply]
np it was found in the numba forums because others was starting to get the same error so one of the devs suggested it as a temporary workaround untill the next version.
